libcloud.groovy

```diff
@@ -30,22 +30,31 @@ def replicate_to_clouds(pipecfg, basearch, buildID) {
     if (meta.amis) {
         replicators["☁️ 🔄:aws"] = {
             def replicated = false
-            def ids = ['aws-build-upload-config', 'aws-govcloud-image-upload-config']
-            def c = pipecfg.clouds.aws
-            for (id in ids) {
-                tryWithCredentials([file(variable: 'AWS_CONFIG_FILE', credentialsId: id)]) {
-                    shwrap("""
-                    cosa aws-replicate \
-                        --build=${buildID} \
-                        --arch=${basearch} \
-                        --credentials-file=\${AWS_CONFIG_FILE} \
-                        --log-level=INFO
-                    """)
-                    replicated = true
+            def aws_info = [[pipecfg.clouds.aws, 'aws-build-upload-config'],
+                            [pipecfg.clouds.aws.govcloud, 'aws-govcloud-image-upload-config']]
+            for (item in aws_info) {
+                def (c, id) = item
+                // Only try to replicate if the config exists
+                if (c) {
```

Comment:
Minor: could invert this and `continue` early to avoid an indentation level.

Reply:
see https://github.com/coreos/fedora-coreos-pipeline/pull/771#issuecomment-1325617342